modules/internal/DefaultRectangular.chpl

```diff
@@ -489,6 +489,11 @@ module DefaultRectangular {
                offset=createTuple(rank, intIdxType, 0:intIdxType))
       where tag == iterKind.follower {
 
+      if followThis.size != this.rank then
+        compilerError("rank mismatch in zippered iteration (leader is " +
+                      followThis.size:string + "D, but array is " +
+                      this.rank:string + "D)");
```

Comment:
Again, I'd like to see the word "follower" to accompany "leader", ex.
`... but array follower is 2D`